waterloop-site/src/pages/Home.tsx

```diff
@@ -34,6 +35,7 @@ const imgJSON: ImgJSON[] = [
 
 const Home: React.FC = () => (
   <div>
+    <NavPopup></NavPopup>
```

Comment:
Looking through some of the code I realized there were a few things I missed. Just make this a singular component `<NavPopup/>`